### ISSUE TITLE
Comment duplication in examples

### ISSUE THREAD

**Issue by chillenzer** [ComputationalRadiationPhysics/picongpu#4813]
While reviewing #4661, I've noticed that all the example input files include all the same explaining comments, in particular all the lists of possible options for solvers, etc. I can see how this is a nice property for examples that are likely used to learn about the usage but at the same time it seems like a lot of maintenance burden to have the same lists in so many places (as can be seen from the endless number of identical changes in #4661). I think the alternative would be to have example specific explanations in the examples and give the extensive lists only in the original `.param` files in the root `include/picongpu/param` folder. The examples could then reference the default `.param` file with respect to that list. Thoughts?

**Comment by BrianMarre** [comment]
good idea, but we should make sure to link them in to readthedocs documentation of each param file to make the additional documentation easier to find.

**Comment by steindev** [comment]
Well, the original idea was to explicitly show the user all available options already in the example. The last time I asked whether we would like to change this, the answer was no :wink:
But from experience of previous PIConGPU workshops, I support this 'no'.

**Comment by chillenzer** [comment]
Well, I am aware of this. And I'm happy to leave it unchallenged... until I have to touch one of those comments for the first time. Then, I'll reopen and you'll have a harder time convincing me.

**Comment by chillenzer** [comment]
But I've got an idea: We could write a pre-commit hook / CI workflow that checks that all those comments are identical. In the best case scenario, it would even update the diverging comments (for example by taking the default one from `include/param/` as ground truth).
